woocommerce/woocommerce-ios
# [Mac Catalyst] 4. Prevent login crash by using WordPressUI with fixed bundle path creation

Review/approve #5659 before this


### Description
When logging in or switching store, there was a 100% crash on Mac Catalyst. This fix adopts the fix in WordPressUI https://github.com/wordpress-mobile/WordPressUI-iOS/pull/106 which resolves the crash. Once that PR is merged and released, this should be updated to use the released version from the pod repo, not a specific branch.



### Testing instructions
1. Build and run for `My Mac`
2. Tap `Continue with WordPress.com` on the first screen, 
3. Enter an email address associated with a gravatar, then continue
4. Note that there is a placeholder briefly displayed before the Gravatar is shown
5. Log in, note that there is no crash.

The same can be repeated using the `⚙️ > Switch Store` option as well.
 



---
